lte/gateway/python/integ_tests/s1aptests/util/traffic_util.py

```diff
@@ -112,55 +113,93 @@ def __init__(self):
         self.ue_ipv6_block = "fdee:0005:006c::/64"
         self.agw_ipv6 = "3001::10"
 
-    def exec_command(self, command):
+    def exec_command(self, command, capture_output=False):
         """
         Run a command remotely on magma_trfserver VM.
 
         Args:
             command: command (str) to be executed on remote host
                 e.g. 'sed -i \'s/str1/str2/g\' /usr/local/bin/traffic_server.py'
+            capture_output: Sets the `capture_output` flag in `subprocess.run`
 
         Returns:
-            Shell command execution output
+            Shell command return code and stdout
         """
-        data = self._cmd_data
-        data["command"] = '"' + command + '"'
-        param_list = shlex.split(self._command.format(**data))
-        return subprocess.call(
+        self._cmd_data["command"] = f'"{command}"'
+        param_list = shlex.split(self._command.format(**self._cmd_data))
+        return subprocess.run(
             param_list,
             shell=False,
-            stdout=subprocess.DEVNULL,
-            stderr=subprocess.DEVNULL,
+            capture_output=capture_output,
         )
 
+    def dump_leases(self):
+        """Dump DHCP leases in TRF server VM"""
+        return self.exec_command("dumpleases", capture_output=True).stdout
+
+    def clear_leases(self):
+        """Remove all DHCP leases in TRF server VM"""
+        cmd = 'systemctl stop udhcpd.service && ' \
+              'rm -f /var/lib/misc/udhcpd.leases && ' \
+              'systemctl start udhcpd.service'
+        return self.exec_command(f"sudo bash -c '{cmd}'").returncode
+
+    @property
+    def n_leases(self):
```

Comment:
NIT: can be private, name: why not count_leases?

Reply:
I used a "n_leases", i.e. a noun because I made this a property so access would to it would be the same as an attribute. I think it would be alright, but I refactored it to just be a regular method `_count_leases`.